css/bs-modal.css

```diff
@@ -0,0 +1,183 @@
+/*!
```

Comment:
I added the bs-modal.css because the classes didn't seem to be present in any of the style sheets currently used in the project.